client/src/app/+admin/overview/users/user-list/user-list.component.ts

```diff
@@ -326,11 +346,14 @@ export class UserListComponent extends RestTable <User> implements OnInit {
   }
 
   protected reloadDataInternal () {
-    this.userAdminService.getUsers({
+    const obs = this.userAdminService.getUsers({
       pagination: this.pagination,
       sort: this.sort,
       search: this.search
-    }).subscribe({
+    })
+    .pipe(shortCacheObservable())
```

Comment:
Is the purpose of `shortCacheObservable` to cache too many calls of `admin-user-list:load-data` in a short period of time)? If yes, prefer to move this to the `addAction` because we don't want the root component to have this side effect, especially since you don't use a cache based on the `getUsers` options.

Reply:
I suggest to remove this cache to avoid side effects, and if it's necessary throttle or debounce the method call

Reply:
The cache was due to multiple API calls made since we had multiple subscribes. I fixed this in the latest commit.